workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSSendSystem.cs

```diff
@@ -89,6 +92,16 @@ private void PopulateDefaultComponentReplicators()
 
                 AddComponentReplicator(componentReplicationHandler);
             }
+
+            // Force the position component to be replicated last to delay authority changes until this frame's updates
+            // are applied as expected. If the position update is applied earlier and triggers an authority change -
```

Comment:
doesn't need a dash. Maybe "A position update can trigger an authority change, which could cause subsequent updates to be dropped".